docs/docs/getting-started/05-examples/03-mock-hardware-example.md

```diff
@@ -87,6 +87,7 @@ buttons:
 components: { }
 hardware:
   mock_hardware:
+    rate: 60
```

Comment:
nitpick: please move rate down below `urdf` just so it looks a bit nicer

Reply:
Done